card/mod/settings/set/type/setting.rb

```diff
@@ -13,7 +13,7 @@ def self.member_names
 
 format :data do
   view :core do
-    wql = { left:  { type: Card::SetID },
+    wql = { left: { type: Card::SetID },
             right: card.id,
             limit: 0 }
```

Comment:
I think we adapted the `def wql_hash` pattern after this was created.  But that could save us some repetition.   We spell out basically identical wql twice here and then another time in wikirate.